docs/guides/postman-collection.md

```diff
@@ -0,0 +1,91 @@
+| oauth2             | http + bearer scheme   |
+| awk                | Hardcoded header value |
+| aws                | Hardcoded header value |
+| akamai             | Hardcoded header value |
+| ntlm               | Hardcoded header value |
+
+## Events
+
+Events are linked to the Postman hosted platform, and so Prism is going to skip this section entirely.
+
+**Warning:** these scripts/event handlers could potentially modify variable values, with the consequence of changing the way the subsequent requests would work.
+
+## Request Body
+
+Postman Collections support saving GraphQL queries as well as regular HTTP. Prism has no GraphQL support at the moment, so that section is going to be skipped.
+
+In case the disabled flag is set to true for any of the parameter type, Prism will ignore such section, as there is no concept of disabled parameter.
+
+## JSON Schema Generation
+
+OpenAPI 2 and 3 are schema driven, while Postman Collections are examples driven.
+
+Essentially, while in OpenAPI 2/3 you can either define an example request/response body OR define a JSON Schema to describe how the payload will look like, in a Postman Collections you can only define examples.
+
+For such reason, Prism will _try_ to infer a JSON Schema from an example, and use such one to perform request/response validation as well as examples generation in case the it is running in `dynamic` mode.
+
+Infer a JSON Schema from a payload is an operation usually complicated — and there's no way a perfect result will be achieved. We can cover the base cases, but keep in mind you might get validation errors that are legit.
+
+To give an example, this JSON object:
+
+```json
+{
+  "name": "Bugatti Veyron",
+  "type": "Sport Car",
+  "speed": 3000
+}
+```
+
+will likely generate this schema:
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string"
+    },
+    "speed": {
+      "type": "number"
+    }
+  }
+}
+```
+
+Although the result seems good:
+
+- There is no way to understand whether all or some of the properties are required or not, so they'll all be marked as optionals.
+- There is no way to understand whether a type is indeed an `enum` instead of a regular `string`. In this specific example, maybe the author's intention is that type is either `Sport Car` or `SUV`
+- There is no way to understand whether a type can be something stricter; for example, maybe the author's intention was to mark the `speed` property as `integer` instead of `number`.
+
+## Items and Folders
+
+Postman Collections have the concept of `items` and `folders` that can be used to better organise your file and give it some structure. Prism will do its best to keep such organisation by fundamentally tagging all the operations with the "folder path", such as `folder1_folder2_folder3_item`. This has no consequences in the CLI experience, but it is worth noting in case you're using the library by yourself.
```

Comment:
@XVincentX that is not true yet. Delete this section + followup issue in http-spec?

Reply:
I've removed it, since this is not even *that* important for Prism users.